tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithRegionTests.cs

```diff
@@ -236,6 +236,25 @@ public async Task RegionFallbackToGlobalAsync()
             }
         }
 
+        [TestMethod]
```

Comment:
nit: you could do ` [TestMethod] [DataRow()]` to have less code. and then it's clear what's being covered.